packages/neo4j-driver-deno/generate.ts

```diff
@@ -130,11 +130,11 @@ async function copyAndTransform(inDir: string, outDir: string) {
       }
 
       // Special fix for bolt-connection/channel/index.js
-      // Replace the "node channel" with the "browser channel", since Deno supports browser APIs
+      // Replace the "node channel" with the "deno channel", since Deno supports browser APIs
```

Comment:
> since Deno supports browser APIs

doesn't make much sense here, I find. Maybe

```suggestion
      // Replace the "node channel" with the "deno channel", since Deno supports different APIs
```